source/assets/public/index.css

```diff
@@ -308,17 +311,67 @@ body {
     margin-top: 2rem;
   }
 
-.back-to-top {
-  text-decoration: none;
-  display: block;
-  width: 10rem;
-  margin-left: auto;
-  margin-right: auto;
+/* Sponsorship section */
+
+.c-sponsor-level__name {
+  font-family: var(--font-heading);
+  color: var(--primary-color);
+  font-size: 3rem;
+  margin-bottom: 2rem;
+  margin-top: 2rem;
 }
-  .back-to-top .body__subtitle {
-    margin-top: 1rem;
-  }
 
-    .back-to-top:hover .body__subtitle {
-      color: var(--primary-color);
-    }
+.c-sponsor-level {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.c-sponsor-level--size-1 .c-sponsor-level__sponsor {
+  flex-basis: 100%;
```

Comment:
I'm not sure what the best way to do this would be, but it may be a good idea to reduce max-dimensions on the sponsor containers - because on larger screen it almost looks comically large (though by the looks of it the dimensions will change when other sponsors are added in so my opinion may change there!